### PR TITLE
Ahelping a "how do I" question now asks if you wish to mhelp it instead

### PR DESCRIPTION
**Pull request by ghost**
:cl: Thefastfoodguy
admin: Ahelping a "how do I" question now asks if you wish to mhelp it instead
/:cl:

### CODE REVIEW

**Comment by KayeArray** [comment]
i'm mixed on this, the 'is this meant for' message is already annoying enough for OOC and having more shit checking widely used phrases is going to get obnoxious

**Comment by tortellinitony** [comment]
I don't like stuff like this

**Comment by ghost** [comment]
nah
answer the question and clarify "game mechanic stuff is better for mentors since admins can see mhelps anyway"
this sorta reeks of passive aggressive with admins getting annoyed by honest questions

**Comment by j-mie** [comment]
I like this, it's only suggestive.

**Comment by mlgdatboi** [comment]
Mentors have a 10% chance of answering your question and its a 5% chance of it being useful
the only real way to get info is via admins or wiki

**Comment by ghost** [comment]
@mlgdatboi admins can see mhelps

**Comment by mlgdatboi** [comment]
neat didnt know that

**Comment by DinduStuffin** [comment]
MLG is wrong, I've been watching mentors like Ivan and Tonythnk take care of mhelps pretty damn well lately.

+1, this seems like a good idea.

**Comment by ghost** [comment]
+1
it being a suggestion is fine, and they can just push it through to the admins instead.

it could possibly allow newer players to be more aware of the existence of the pink menace.

File: code/modules/admin/verbs/adminhelp.dm [hunk 1]
```diff
@@ -493,6 +493,15 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 
 	if(!msg)
 		return
+
```
Comment (j-mie) [inline code/modules/admin/verbs/adminhelp.dm:496]: make a proc in the hippie folder and call it, don't add extra code.

**Review by j-mie — CHANGES_REQUESTED** [review]

**Review by j-mie — DISMISSED** [review]

**Comment by ghost** [comment]
build this

**Comment by Super3222** [comment]
idk if admins can interact with mentor helps, but if not you should make this check to see if there are mentors online.

**Pull request closed without merging**